diff --git a/src/main.cpp b/src/main.cpp
index 5373cce4..75b13fed 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -87,7 +87,6 @@ extern "C" int Q_DECL_EXPORT kdemain(int argc, char *argv[])
 {
     // enable high dpi support
     QCoreApplication::setAttribute(Qt::AA_UseHighDpiPixmaps, true);
-    QCoreApplication::setAttribute(Qt::AA_EnableHighDpiScaling, true);
 
     // Check if any of the arguments makes it impossible to re-use an existing process.
     // We need to do this manually and before creating a QApplication, because
