+++ b/applets/taskmanager/package/contents/ui/Task.qml	2016-02-14 23:10:55.298230704 -0700
@@ -257,7 +257,7 @@
             visible: false
 
             active: task.containsMouse
-            enabled: true
+            enabled: !(model.Minimized && !plasmoid.configuration.showOnlyMinimized)
 
             source: model.DecorationRole
 
@@ -312,7 +312,7 @@
 
         visible: (inPopup || !iconsOnly && !model.IsLauncher && (parent.width - LayoutManager.horizontalMargins()) >= (theme.mSize(theme.defaultFont).width * 7))
 
-        enabled: true
+        enabled: !model.Minimized
 
         text: (!inPopup && iconsOnly) ? "" : model.DisplayRole
         color: theme.textColor
